libs/auth/scripts/create-production-machine-cert-signing-request

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
+
+require('esbuild-runner/register');
+require('./create_production_machine_cert_signing_request').main();
```

Comment:
I actually use this script in https://github.com/votingworks/vxsuite-complete-system/pull/302/commits/a1b5c379fbe95d659b1bf1482fd0b7f40cbde405